crates/binswap-github/Cargo.toml

```diff
@@ -12,13 +12,12 @@ repository = { workspace = true }
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-binstalk = "0.8.0"
+binstalk = { version = "0.12.0", features = ["trust-dns"] }
```

Comment:
Can you comment on why `trust-dns` is used? To be clear, I'm no way against it, but I'd like to hear the benefit.

Reply:
I enabled feature `binstalk/trust-dns` to use trust-dns instead of libc's `getaddrinfo`, which is just often broken (previously in musl libc, recently fixed by adding tcp dns resolve support) and should not be used in any modern application.

Reply:
Thanks! I wasn't aware of this.